charger/tasmota.go

```diff
@@ -134,6 +134,19 @@ func (c *Tasmota) CurrentPower() (float64, error) {
 		power = 0
 	}
 
+	// set fix static power in static mode
+	if c.standbypower < 0 {
```

Comment:
Wäre es nicht schöner, das so zu schreiben, dass nur die notwendigen API-Aufrufe gemacht werden? In einem Fall brauchst Du nur Power, im Anderen auch mal nur Enabled? Ich weiss allerdings nicht, obs elegant geht.